packages/lab/src/stepper/Step.Connector.css

```diff
@@ -0,0 +1,52 @@
+.saltStepConnector {
+  --saltStep-connector-borderWidth: 2px;
```

Comment:
--salt-size-border-strong ?

Reply:
I think that one is deprecated, I remember one of them complaining

Reply:
No, it's not deprecated, placing it right away!